src/Microsoft.HttpRepl.IntegrationTests/Commands/GetCommandTests.cs

```diff
@@ -0,0 +1,176 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.HttpRepl.Fakes;
+using Microsoft.HttpRepl.IntegrationTests.SampleApi;
+using Microsoft.HttpRepl.IntegrationTests.Utilities;
+using Xunit;
+
+namespace Microsoft.HttpRepl.IntegrationTests.Commands
+{
+    public class GetCommandTests : BaseIntegrationTest, IClassFixture<HttpCommandsFixture<SampleApiServerConfig>>
+    {
+        private readonly SampleApiServerConfig _serverConfig;
+
+        public GetCommandTests(HttpCommandsFixture<SampleApiServerConfig> fixture)
+        {
+            _serverConfig = fixture.Config;
+        }
+
+        [Fact]
+        public async Task WithoutParameter_ShowsColorizedOutput()
+        {
+            string scriptText = $@"set base {_serverConfig.BaseAddress}
+cd api/values
+get";
+
+            var console = new LoggingConsoleManagerDecorator(new NullConsoleManager());
+            using (var script = new TestScript(scriptText))
+            {
+                await new Program().Start($"run {script.FilePath}".Split(' '), console);
+            }
+
+            string output = console.LoggedOutput;
+            // remove the first line because it has the randomly generated script file name.
+            output = output.Substring(output.IndexOf(Environment.NewLine) + Environment.NewLine.Length);
+            output = NormalizeOutput(output, _serverConfig.BaseAddress);
+
+            string expected = NormalizeOutput(@"(Disconnected)~ set base [BaseUrl]
+Using swagger metadata from [BaseUrl]/swagger/v1/swagger.json
+
+[BaseUrl]/~ cd api/values
+/api/values    [get|post]
+
+[BaseUrl]/api/values~ get
+[32m[1mHTTP[22m[39m[32m[1m/[22m[39m[32m[1m1.1[22m[39m [33m[1m200[22m[39m [33m[1mOK[22m[39m
```

Comment:
I'm assuming the gibberish here are the console color codes.  If I have alternate colors configured, will it affect running tests on my machine?  Should we (or even can we) normalize these as well?

(For that matter, will we eventually have integration tests for configuring colors...)

Reply:
Yes, your configuration would impact these tests as far as I can tell. Normalizing it might be tricky... the same color can be from different sources, so it's not really reversible. 

I _would_ like to have integration tests for configuring the colors, especially since in some cases there's no color codes in the output (if you've specified None). So the output looks different than the default both raw and rendered. 

I'll ponder that a little bit more - we need a solution since the colors are going to be in everything that has output from a server, plus errors and warnings.

Reply:
One option might be to have a set of preferences that we apply during tests to normalize the output (whether part of the test script, or an injected `IPreferences`).  That at least alleviates the variability based on local settings.

Aside from tests actually around color settings, I wonder if we'd lose anything by setting all colors to none (i.e. preventing getting these codes in the output).  It would mean that tests assert the right textual output, but no longer enforce the right colorization.  On a select few tests, we can re-enable colors to make sure we're doing the right thing.  Thoughts?

Reply:
I thought about your second paragraph a good bit last night and I just can't really decide. The colors are kind of an intrinsic part of the output - whether it is the json colorization of output or warning/error colorization of messages.

Maybe there's a middle ground where we keep the error/warning colors, since those will pollute the output a lot less but make all the others None? The error/warning colors are truly part of the message in my mind... but the json colorization (or xml colorization in the future) is just a readability helper. But that might be a bit of post-decision reduction on my part.